ostd/src/arch/x86/iommu/dma_remapping/context_table.rs

```diff
@@ -51,18 +51,23 @@ pub enum ContextTableError {
 }
 
 impl RootTable {
-    pub fn new() -> Self {
+    pub fn root_paddr(&self) -> Paddr {
+        self.root_frame.start_paddr()
+    }
+
+    pub(super) fn new() -> Self {
         Self {
             root_frame: FrameAllocOptions::new(1).alloc_single().unwrap(),
```

Comment:
Although not quite fitting in the scope of this PR, I think that the root table is still typed memory, and should be represented by a `Page`. Again, I am OK with not introducing such change in this PR.